lbrycrd/stuff/start.sh

```diff
@@ -6,35 +6,39 @@
 # ## not specified it will only create an index for transactions that are related to the wallet or have unspent outputs.
 # ## This is specific to chainquery.
 
+# The config file does not exist in the container image. It must be mounted, or
+# if not, a default config is generated using environment variables.
+CONFIG_PATH=/etc/lbry/lbrycrd.conf
+if [ -f "$CONFIG_PATH" ]
+then
+    echo "Using the config file mounted into the container"
+else
+    echo "Creating fresh config file from environment variables"
+    ## Set config params
+    mkdir -p `dirname $CONFIG_PATH`
+    echo "rpcuser=$RPC_USER" > $CONFIG_PATH
+    echo "rpcpassword=$RPC_PASSWORD" >> $CONFIG_PATH
+    echo "rpcallowip=$RPC_ALLOW_IP" >> $CONFIG_PATH
+    echo "rpcport=9245" >> $CONFIG_PATH
+    echo "rpcbind=0.0.0.0" >> $CONFIG_PATH
+    #echo "bind=0.0.0.0" >> $CONFIG_PATH
+fi
+
```

Comment:
Gosh I forgot that I'd not done this for lbrycrd my priority at the time was Chainquery.  Thank you for doing this.